docs/API.md

```diff
@@ -113,6 +114,7 @@ Returns a new [logger](#logger) instance.
   message. Default: `msg`.
   * `forceColor` (boolean): if set to `true`, will add color information to the formatted output
   message. Default: `false`.
+  * `crlf` (boolean): emit `\r\n` instead of `\n`. Default: `false`.
```

Comment:
This language should match the language in the options for the regular constructor.

Reply:
I changed the wording above to

> logs newline delimited JSON with `\r\n` instead of `\n`. Default: `false`.

as suggested by @davidmarkclements, but this is not necessarily true for `pretty` (which can produce arbitrary strings, not just JSON), right?

Reply:
Good call. Ignore me.